platform/orion/driver/committer.go

```diff
@@ -9,9 +9,14 @@ package driver
 import (
 	"context"
 
+	"github.com/hyperledger-labs/fabric-smart-client/platform/common/core/generic/committer"
 	"github.com/hyperledger-labs/fabric-smart-client/platform/common/driver"
 )
 
+type ListenerManagerProvider = committer.ListenerManagerProvider[ValidationCode]
```

Comment:
driver should not depend on core. Maybe you want to move those interfaces under `common/driver`.